PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutInternalClient.java

```diff
@@ -62,10 +62,9 @@ void sendRequest(final Context context, final PayPalNativeRequest payPalRequest,
                                         String clientMetadataId = payPalRequest.getRiskCorrelationId() != null
                                                 ? payPalRequest.getRiskCorrelationId() : payPalDataCollector.getClientMetadataId(context, configuration, payPalRequest.hasUserLocationConsent());
 
+                                        payPalResponse.clientMetadataId(clientMetadataId);
                                         if (pairingId != null) {
-                                            payPalResponse
-                                                    .pairingId(pairingId)
-                                                    .clientMetadataId(clientMetadataId);
+                                            payPalResponse.pairingId(pairingId);
```

Comment:
PayPalWeb and PayPalNative share the same text fixtures (and same implementation for the most part). This change mirrors the one [made in PayPalWeb](https://github.com/braintree/braintree_android/pull/983/files#r1588466622).